src/_scss/components/_pagination.scss

```diff
@@ -1,6 +1,6 @@
 .pagination {
     margin: rem(15);
-    .results-text {
+    .pagination__totals {
         @include display(flex);
         @include flex-direction(row);
         float: left;
```

Comment:
This `float: left` and the next `float: right` are a weird mix of flexbox and floating. Can you have the parent block be a flexbox, with `totals` and `pager` being flex items within that container that, in turn, act as flex containers for their children?

Reply:
Removed `float`s and added flexbox properties